apache2/waf_logging/waf_log_util.cc

```diff
@@ -195,7 +199,7 @@ static std::string get_json_log_message(const char* resource_id, const char* ope
                     details_messages,
                     details_data_str,
                     details_file_str,
-                    details_line_str
+                    details_line_str,
```

Comment:
If no new object is passed after `details_line_str`, please revert this line change to reduce the amount of unnecessary modifications.